### ISSUE TITLE
Add multilingual option

### ISSUE THREAD

**Issue by MusikAnimal** [MusikAnimal/pageviews#11]
Say if I wanted to see the page views of Google across all languages of the current project. Options might allow you to specify specific languages (or projects, I guess) or just show all languages.

This could be expensive, not sure. Either way we'll have to rework our local version of the site matrix.

**Comment by MusikAnimal** [comment]
See also https://tools.wmflabs.org/hay/langviews/ which still relies on stats.grok.se

**Comment by kaldari** [comment]
Hmm. Seems like option cruft to me as I can't think of many cases where this would actually be needed and it would require a significant expansion of the UI (if you wanted to choose specific languages to compare). Perhaps this could be part of a advanced version of the UI (that is normally hidden). Personally, I really like how clean the current UI is and I hope it doesn't get too cluttered with UI for obscure edge cases.

**Comment by MusikAnimal** [comment]
Yep, my thoughts are the same. I was thinking it'd a different view entirely, perhaps dubbed "Langviews", routed at `/pageviews/langviews`
